code/modules/mob/living/emote.dm

```diff
@@ -11,17 +11,11 @@
 
 /datum/emote/living/bow
 	key = "bow"
-	key_third_person = "bows"
+	key_third_person = "bowss"
```

Comment:
This doesn't seem right

Reply:
oh yeah, that, that was me testing if comitting worked hahaha